### ISSUE TITLE
Playlist 'trackCount' returning unexpected value

### ISSUE THREAD

**Issue by JohnHKoh** [sigma67/ytmusicapi#365]
When using the `YTMusic.get_playlist` method ([documentation](https://ytmusicapi.readthedocs.io/en/latest/reference.html#ytmusicapi.YTMusic.get_playlist)), getting the `trackCount` value from the returned playlist object seems to return an unexpected value. This has only recently started happening.

For example, the playlist ID "PLnlxKMP5GzKCXJ3Cw99qSWgC01cuhTFxG" has 100 songs. However, `playlist['trackCount']` returns 13, while `len(playlist['tracks'])` returns the correct value of 100. This issue first started appearing on March 22, 2023. Before then, `trackCount` would return the same value as the length of `tracks`, as expected. I've tried multiple playlist, and none of the `trackCount` values match up with the number of songs in the playlist. Did YouTube update what `trackCount` means?

**Comment by mtbrandao** [comment]
Looking at the web interface, looks like the playlists have a new field called "views".
Example: "7.1K views"

For me, the "trackCount" value matches the "view" value on the web interface

**Comment by mtbrandao** [comment]
The "get_playlist" function also uses the "trackCount" value to orient itself.
As currently, the track count is wrong, the "get_playlist" function is only returning 100 song maximum.

**Comment by JohnHKoh** [comment]
> Looking at the web interface, looks like the playlists have a new field called "views". Example: "7.1K views"
> 
> For me, the "trackCount" value matches the "view" value on the web interface

This is exactly it. I've tried multiple playlists. If it says "8.5K views", `trackCount` will return 85. If it says "22 views", `trackCount` will return 22. It seems to be the numeric part of the view count without the decimal.

> I'm not able to reproduce. For me the playlist you provided has a `trackCount` of 100 and 100 returned playlist items. It's a top 100 playlist, so that's exactly what's expected.

I've figured it out. This bug seems to only happen on playlists that you have created. If you use someone else's playlist, `trackCount` will use the right value. This is because the view count is only shown when viewing your own playlist. Look at the difference when viewing the same playlist in different scenarios:

Logged in:
![image](https://user-images.githubusercontent.com/6465531/227643198-31065dd1-3dfb-47fa-acea-8a78de6c677d.png)

Logged out (incognito):
![image](https://user-images.githubusercontent.com/6465531/227643264-ef4ae745-8d24-4c76-906d-6e0446d2f352.png)

It seems `trackCount` is taking the first value there.
